about.html

```diff
@@ -0,0 +1,123 @@
+      <nav class="nav-bar">
+        <ul>
+          <li><a href="index.html" class="nav-link" id="Home">Home</a></li>
+          <li><a href="#" class="nav-link">About</a></li>
+          <li><a href="#Program" class="nav-link">Program</a></li>
+          <li><a href="#Geusts" class="nav-link" >Guests</a></li>
+          <li><a class="nav-link" href="#Parteners">Partners</a></li>
+          <li><a href="#Envents" class="nav-link">Event</a></li>
+          <li><a href="#" class="nav-link"><span id="mawazine">mawazine</span> </a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <div class="head">
+      <p class="slogan">"Largest Music Festival in Africa"</p>
+      <h1 class="title">
+        Mawazine Music <br>
+      <span class="title-content">Festival</span> <br>
+      <span class="title-content">2022</span></h1>
+    </div>
+    <div class="description">
+      <p>The Mawazine Music Festival is the largest conference dedicated to music in the Africa. Held every year in Rabat Morroco, the conference features speakers from the music industry, musicians, and music education professionals. This year’s conference will be held on July 25-27 at the same place.</p>
+    </div>
+    <div class="contact">
+      <p>Please contact us per Email for any further questions about Mawazine Music Festifal 2022!</p>
+      <a href="#">mawazinerabat@gmail.com</a>
+    </div>
+  </main>
+  <section class="log-mmf">
+    <div class="section-title">
+      <h2 >Mawazine Music Festifal</h2>
+      <span class="bottom-title"></span>
+    </div>
+     <p>
+      The logo for the Mawazine Music Festival 2022 was decided through the
+      logo competition from 21st to 22nd of July this year. After online
+      public voting and Selection Committee's judging process,the logo from
+      <strong>Maaloum Ely Cheick</strong> won the price.
+    </p>
+    <div class="logodegn">
+      <img src="Assets/mawazine.png" alt="logo">
+    </div>
+  </section>
+  <section class="previous-event">
+    <div class="section-title">
+      <h2 >See Past MMF Events</h2>
+      <span class="bottom-title"></span>
+      <p>Take a look at the last two Mawazine Music Festivals which took place in Casablanca and Nouakchott.</p>
+    </div>
+    <ul class="past-events">
+      <li>
+        <h3> 2020 <span>Mawazine Music Festival</span></h3>
+      </li>
+      <li>
+        <h3> 2021 <span>Mawazine Music Festival</span></h3>
+      </li>
+    </ul>
+  </section>
+  <section class="partners-sec" id="Parteners">
+    <div class="section-title">
+      <h2>Panters</h2>
+      <span class="bottom-title"></span>
+     </div>
+
+    <div class="partners">
+      <img src="Assets/mtc-logo.png" alt="partner1" >
+      <img src="Assets/anghami.png" alt="partner2" >
+      <img src="Assets/pepwear-logo.png" alt="partner3" >
+      <img src="Assets/coke-studio-logo.png" alt="partner4" >
+      <img src="Assets/yamaha-logo.png" alt="partner5" >
+    </div>
```

Comment:
- [ ] [OPTIONAL] - Kindly note that this section according to the Behance should be hidden in the desktop version. Kindly consider making this change to hide it on desktop view.

**Your design:**

![image](https://user-images.githubusercontent.com/86318284/182801810-5cadaf10-764f-4307-97bc-bc854b4493df.png)

**Desired Behance design:**

![image](https://user-images.githubusercontent.com/86318284/182801909-84fd9a3b-6082-4c0a-a8f8-886fe57b40b4.png)